Fix compile error for Spotify search endpoint

```
src/routes/spotify_routes.ts:79:51 - error TS2345: Argument of type 'string | string[] | ParsedQs | ParsedQs[]' is not assignable to parameter of type 'string | number | boolean'.
  Type 'string[]' is not assignable to type 'string | number | boolean'.
    Type 'string[]' is not assignable to type 'string'.

const encodedQuery: string = encodeURIComponent(req.query.q);
```

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "music-club",
-  "version": "1.8.0",
+  "version": "1.8.1",
   "description": "Music Club backend service",
   "main": "dist/server.js",
   "scripts": {

--- a/src/routes/spotify_routes.ts
+++ b/src/routes/spotify_routes.ts
@@ -76,7 +76,7 @@ router.get('/api/album-search', async (req: Request, res: Response): Promise<voi
     return;
   }
 
-  const encodedQuery: string = encodeURIComponent(req.query.q);
+  const encodedQuery: string = encodeURIComponent(req.query.q.toString());
 
   const requestFun = async (): Promise<void> => {
     const searchResult: any = await axios({

